docs/java-rest/high-level/apis/search.asciidoc

```diff
@@ -21,7 +21,7 @@ which contains everything that
 in the Rest API would be placed in the search request body.
 <3> Add a `match_all` query to the `SearchSourceBuilder`.
 
-==== Optional arguments
+===== Optional arguments
 
 Lets first look at some of the optional argument of a `SearchRequest`.
 First of all, the request can be restricted to one or more indices using the
```

Comment:
This sentence doesn't make sense , sorry I didn't catch that earlier: "using the constructor or to on or more types using a setter"